bug_update.php

```diff
@@ -364,6 +364,7 @@
 
 # Handle automatic assignment of issues.
 if( $t_existing_bug->handler_id === NO_USER &&
+         $t_updated_bug->status == $t_existing_bug->status &&
```

Comment:
Adjust indentation to match neighboring lines.  Move this check to be the first or the third check, since the two checks for handler makes sense to be next to each other.